azure-iot-device/azure/iot/device/common/error_map.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from . import errors
+
+status_code_to_error = {
```

Comment:
>status_code_to_error [](start = 0, length = 20)

oh never mind the second part of my comment on the errors.py file. You already have this dictionary here.